tests/unit-ext/test-ext-arg.c

```diff
@@ -489,6 +493,77 @@ test_validator_int3_handler (const jerry_value_t func_obj_val __attribute__((unu
 } /* test_validator_int3_handler */
 
 static jerry_value_t
+test_validator_array1_handler (const jerry_value_t func_obj_val __attribute__((unused)), /**< function object */
+                               const jerry_value_t this_val __attribute__((unused)), /**< this value */
+                               const jerry_value_t args_p[], /**< arguments list */
+                               const jerry_length_t args_cnt __attribute__((unused))) /**< arguments length */
+{
+  double native1 = 0;
+  double native2 = 0;
+  double native3 = 0;
+
+  jerryx_arg_array_props_t arr_info;
+
+  jerryx_arg_t item_mapping[] =
+  {
+    jerryx_arg_number (&native1, JERRYX_ARG_COERCE, JERRYX_ARG_REQUIRED),
+    jerryx_arg_number (&native2, JERRYX_ARG_COERCE, JERRYX_ARG_REQUIRED),
+    jerryx_arg_number (&native3, JERRYX_ARG_COERCE, JERRYX_ARG_OPTIONAL)
+  };
+
+  arr_info.c_arg_p = item_mapping;
+  arr_info.c_arg_cnt = 3;
+
+  jerryx_arg_t mapping[] =
+  {
+    jerryx_arg_array (&arr_info, JERRYX_ARG_OPTIONAL),
+  };
+
+  jerry_value_t is_ok = jerryx_arg_transform_args (args_p, 1, mapping, 1);
```

Comment:
Is `(args_p, 1,` OK? Shouldn't this be `(args_p, args_cnt,`. I see that we have a similar construct in `test_validator_prop2_handler`, but actually that does not seem right to me either.

Reply:
I will change them all and avoid the magic number, thanks!

Reply:
I will add a macro to get the item count of the mapping
```
#define ARRAY_SIZE(array) ((unsigned long)(sizeof(array) / sizeof((array)[0])))
```